packages/next/client/use-intersection.tsx

```diff
@@ -16,14 +23,15 @@ type Observer = {
 const hasIntersectionObserver = typeof IntersectionObserver !== 'undefined'
 
 export function useIntersection<T extends Element>({
+  rootEl,
```

Comment:
Let's call this `rootRef` to be clear its not a HTMLElement but a RefObject

Reply:
Done